### PR TITLE
HPCC-10762 Display Graph Start/Finish times

### PR DESCRIPTION
**Pull request by GordonSmith**
Display start and finish times (if available).

Fixes HPCC-10762

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/GraphsWidget.js [hunk 1]
```diff
@@ -128,8 +128,28 @@ define([
...
+                            if (whenFinished) {
+                                var dateTime = new Date(whenFinished);
+                                return dateTime.toLocaleTimeString();
+                            }
+                            return "";
+                        }
+                    },
                     Time: {
-                        label: this.i18n.Time, width: 90, sortable: true,
+                        label: this.i18n.Duration, width: 90, sortable: true,
```
Comment (miguelvazq) [inline esp/src/eclwatch/GraphsWidget.js]: Isn't there still an issue with assigning widths to all columns?

**Comment by miguelvazq** [comment]
Review complete see comment above.

File: esp/src/eclwatch/GraphsWidget.js (hunk 1 above)
Comment (GordonSmith) [inline esp/src/eclwatch/GraphsWidget.js]: Label doesn't have  width (on line 129)

**Comment by miguelvazq** [comment]
Missed that. Looks good.

**Pull request merged**